src/VisualStudio/Core/Impl/RoslynVisualStudioWorkspace.cs

```diff
@@ -29,16 +30,19 @@ namespace Microsoft.VisualStudio.LanguageServices
     internal class RoslynVisualStudioWorkspace : VisualStudioWorkspaceImpl
     {
         private readonly IEnumerable<Lazy<IStreamingFindUsagesPresenter>> _streamingPresenters;
+        private readonly IEnumerable<Lazy<ISymbolicNavigationService>> _symbolicNavigationServices;
 
         [ImportingConstructor]
         private RoslynVisualStudioWorkspace(
             ExportProvider exportProvider,
             [ImportMany] IEnumerable<Lazy<IStreamingFindUsagesPresenter>> streamingPresenters,
             [ImportMany] IEnumerable<IDocumentOptionsProviderFactory> documentOptionsProviderFactories,
-            [Import(typeof(SVsServiceProvider))] IAsyncServiceProvider asyncServiceProvider)
+            [Import(typeof(SVsServiceProvider))] IAsyncServiceProvider asyncServiceProvider,
+            [ImportMany] IEnumerable<Lazy<ISymbolicNavigationService>> symbolicNavigationServices)
```

Comment:
Nitpick: let's keep it up by the streaming presenters since that's often being passed around as a pair.

Reply:
Done.